solvers.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "solvers",
+  "buildsystem": "simple",
+  "build-commands": [],
+  "modules": [
+    {
+      "name": "rinutils",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/shlomif/rinutils/releases/download/0.10.1/rinutils-0.10.1.tar.xz",
+          "sha256": "31ec258ce99de7bbbe79f3338ce73048dac45425045eb2bd0d6bd92d1b8b9afb",
+          "x-checker-data": {
+            "type": "html",
+            "url": "https://github.com/shlomif/rinutils/releases/latest",
+            "version-pattern": "https://github.com/shlomif/rinutils/releases/expanded_assets/([\\d\\.]+\\d)",
+            "url-template": "https://github.com/shlomif/rinutils/releases/download/$version/rinutils-$version.tar.xz"
+          }
+        }
+      ]
+    },
+    {
+        "name": "solvers_extra_deps",
+        "buildsystem": "simple",
+        "build-commands": [
+            "solvers_extra_deps/install.sh"
+        ],
+        "post-install": [
+            "chmod -Rf u+w /app/lib/perl5/site_perl"
+        ],
+        "sources": [
+            "solvers_extra_deps.json"
+        ]
+    },
+    {
+      "name": "fc-solve",
+      "buildsystem": "cmake",
+      "build-options": {
+          "env": {
+              "PERL5LIB": "/app/lib/perl5/site_perl/5.36.0"
+          }
+      },
+      "config-opts": [ "-DFCS_WITH_TEST_SUITE=OFF" ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://fc-solve.shlomifish.org/downloads/fc-solve/freecell-solver-6.6.0.tar.xz",
+          "sha256": "16b58e78d44d95d0ff1b10496dd2d8bd27de592a26f8f874e95a1d841309048c",
+          "x-checker-data": {
+            "type": "html",
+            "url": "https://fc-solve.shlomifish.org/download.html",
+            "pattern": "(downloads/fc-solve/freecell-solver-([\\d\\.]+\\d).tar.xz)"
+          }
+        }
+      ]
+    },
+    {
+      "name": "black-hole-solitaire-solver",
+      "buildsystem": "cmake",
+      "build-options": {
+          "env": {
+              "PERL5LIB": "/app/lib/perl5/site_perl/5.36.0"
```

Comment:
did you try running the package?

Perl5 is only in the SDK. So you need to build it.

Reply:
ah ok

Reply:
Yes. Aside from originally forgetting to modify the `.desktop` file to use the wrapper, and thus missing that music only worked when launching from the command line, I tested everything. The solvers work when run like this:

```sh
flatpak-builder --user --install --force-clean build io.sourceforge.pysolfc.PySolFC.json
flatpak run io.sourceforge.pysolfc.PySolFC
```

As I remember, Perl is used in the build process but is not a runtime dependency. (The build process for a PySolFC with all optional features enabled is a bit odd at times.)

Reply:
To elaborate on that, PySolFC uses Perl for unit testing and code checking during the build process.

Reply:
I didn't see it cleaned up though.

Reply:
Probably an oversight. The solvers were a bit of a slog to get building with an immutable `/usr` and I was less attentive as a result.

I'll add it to cleanup and test in a moment.

Reply:
Yeah. Everything still works with that cleaned up.